yinstruments/pdu/main.py

```diff
@@ -0,0 +1,55 @@
+from netbooter import Netbooter
+from lindy import Lindy
+import os
+import argparse
+
+def print_usage():
+    print("Incorrect usage!")
+    print(os.path.splitext(__file__)[0], "<on|off|reboot|status> <port_num>")
+
+
+def main():
+    #create instance of argparse
+    arguments = argparse.ArgumentParser(description="Command Line Arguments")
+    arguments.add_argument('dev_type', type=str, help = "DEVELOPER TYPE: What brand of pdu you are communicating with --> 1st command line argument after file name")
+    arguments.add_argument('ip_address', type=str, help = "IP ADDRESS: IP of your pdu --> 2nd command line argument")
+    arguments.add_argument('command', type = str, help = "COMMAND: A string of what port number you would like to call a function on --> 3rd command line argument")
```

Comment:
Change help to "string of command type to issue to PDU"

Add `choices=("on", "off", "reboot", "get_status")`